src/components/HotspotListItem.tsx

```diff
@@ -53,7 +54,10 @@ const HotspotListItem = ({
             </Text>
           </Box>
           <Text variant="body2" color="purpleMain" paddingTop="s">
-            {`+${totalReward.toString(2)}`}
+            {`+${totalReward.toString(2, {
```

Comment:
could maybe encapsulate the i18n part of this into the currency util file?

Reply:
Yeah its just that we do use other options too, like sometimes we want 8 decimals and sometimes we wanted fixed 2 or just rounded 2. Its a little tough to consolidate it more without ending up with something pretty similar